### PR TITLE
runtime function apps

### PR DESCRIPTION
**Pull request by JosiahSiegel**
add 2 runtime based function apps and a common module for easier creation

### CODE REVIEW

**Review by DanPaseltiner — APPROVED** [review]
This looks good to me.

File: operations/app/terraform/modules/storage/data.tf [hunk 1]
```diff
@@ -25,6 +23,10 @@ locals {
     { permissions = "rwx", id = var.adf_uuid, type = "user", scope = "default" },
     { permissions = "rwx", id = var.pdi_function_app_uuid, type = "user", scope = "access" },
     { permissions = "rwx", id = var.pdi_function_app_uuid, type = "user", scope = "default" },
+    { permissions = "rwx", id = var.java_function_app_uuid, type = "user", scope = "access" },
```
Comment (cwinters-usds) [inline operations/app/terraform/modules/storage/data.tf:26]: nice to see how these ACLs tie into the function apps :100:

File: operations/app/terraform/vars/test/variables.tf [hunk 2]
```diff
@@ -98,11 +98,11 @@ variable "dns_vars" {
 ##################
 
 variable "app_tier" {
-  default = "PremiumV2"
+  default = "PremiumV3"
```
Comment (cwinters-usds) [inline operations/app/terraform/vars/test/variables.tf:101]: curious what prompted this change?

**Review by cwinters-usds — APPROVED** [review]
couple of questions just cause I'm curious but :tada:

File: operations/app/terraform/vars/test/variables.tf (hunk 2 above)
Comment (JosiahSiegel) [reply operations/app/terraform/vars/test/variables.tf:101]: The data ingest team made that change in their "test/prod" environment, so I update the TF to match.

**Pull request merged**